### PR TITLE
Re-enable cookie banner tests for test env

### LINKED ISSUE #1937: Re-enable WS cookie banner integration tests
Blocking on routing to STMs

**Is your feature request related to a problem? Please describe.**
In https://github.com/bbc/simorgh/pull/1938, the World Service cookie banner integration tests were restricted to run on local env only, as the routing was not in place on our test environment. When the routing is in place, we should allow these tests to be ran on local **and** test environments.
 
**Describe the solution you'd like**
Allow these tests to be ran on local and test env's:
https://github.com/bbc/simorgh/blob/61d29a2729e372c7d8c75b4f24d93691451d5539/cypress/integration/cookieBannerWS.js#L13-L21

https://github.com/bbc/simorgh/blob/61d29a2729e372c7d8c75b4f24d93691451d5539/cypress/integration/cookieBannerWS.js#L27-L35
**Testing notes**
[Tester to complete]

Dev insight: Enabled cypress tests should pass

Make sure @david-boydell sees this before merge as he'll be iterating the tests in future

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #1937 

**Overall change:** _Re-enable cookie banner integrations tests for test environment._

**Code changes:**

- _Run integration test in `Local` and `Test` environments._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by Bopchy — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by david-boydell** [comment]
I'm going to block this in Ready for Test for the moment whilst I do some investigation.

It does not seem like articles data is available on the test environment, so I'm unsure exactly what the parent issue is requesting.

**Comment by david-boydell** [comment]
This is blocked awaiting https://github.com/bbc/simorgh-infrastructure/issues/353

**Comment by jamesbhobbs** [comment]
There's no need to run the cookie tests on an article it can be on any page. Far easier to do it on front pages. I've done that and excluded the broken services here: https://github.com/bbc/simorgh/pull/2718

File: cypress/integration/cookieBannerWS.js [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
+import { describeForLocalAndTest } from '../support/limitEnvRuns';
 import services from '../../src/app/lib/config/services';
 
 Object.keys(services).forEach(index => {
   const serviceConfig = services[index];
   const service = index;
 
   // This should be unhacked when this file is consolidated with other cookie testing files.
-  if (service === 'news' || service === 'default') {
+  if (
```
Comment (dr3) [inline cypress/integration/cookieBannerWS.js]: Why cant we use a key in the cypress service config for this? This style of if X exception is something we should reeeeallly avoid doing IMO

**Review by dr3 — CHANGES_REQUESTED** [review]

File: cypress/integration/cookieBannerWS.js (hunk 1 above)
Comment (jamesbhobbs) [reply cypress/integration/cookieBannerWS.js]: I don't mind either way, my change was just proving that it's easy to fix.

Comment (david-boydell) [reply cypress/integration/cookieBannerWS.js]: I can stick in a `isWorldService` into the cypress/support/config/services.js file. Not sure that's the best place for the services that are not yet on test though.

How do you feel @benjaminhobbs @dr3 about amending the cypress/support/config/envs.js file to list the locales that a particular environment has or does not have, in an array.

If that suits you both, shall I go for a list of ones that they **do not** have - that way it's an ever reducing list that can eventually be removed.

Comment (david-boydell) [reply cypress/integration/cookieBannerWS.js]: e.g.
``` 
test: {
    baseUrl: 'https://www.test.bbc.com',
    dataUrl: 'https://www.test.bbc.com',
    assetUrl: 'https://news.test.files.bbci.co.uk/include/articles/public',
    assetOrigin: 'https://news.test.files.bbci.co.uk',
    atiAnalyticsWSBucket: '598343',
    atiUrl: 'https://logws1363.ati-host.net?',
    lacksLocale: ['serbian', 'telugu', 'ukchina', 'zhongwen'],
  },
```

Comment (david-boydell) [reply cypress/integration/cookieBannerWS.js]: I'll be honest I think we should merge this as is and tackle the refactor when we merge the tests. Doing it bit by bit will take too long.

The tests use various sources - some seem to use use src/app/lib/config/services/... as the driver, others use cypress/support/config/services.js.

I think a more consistent approach will mean we can leverage helpers further down the line. The tests works as expected so best to get these merged in.

**Comment by david-boydell** [comment]
Hi @dr3, it lookslike Benjamin has already created a ticket for the refactor of this (https://github.com/bbc/simorgh/issues/2215). I was going to pick it up but it's assigned to Olu.

**Review by dr3 — APPROVED** [review]

**Comment by david-boydell** [comment]
Thanks Einstein, this is ready for merge.

**Pull request merged**